### PR TITLE
Support IPv6 offloading and provide throughput benchmarking tools

### PR DESCRIPTION
**Pull request by byteocean**
This PR contains modifications to enable IPv6 tunnel offloading, including VM2VM on the same hypervisor, VM2VM across the hypervisors, and LB setup. It also provides newly created throughput testing tools and corresponding results under different scenarios.

### CODE REVIEW

**Review by PlagueCZ — COMMENTED** [review]
I have tried on my machine, but the client only establishes the connection  (Ping successful and TCP connetion established) and does nothing more. I may be doing something wrong, but I did use arguments in the docs

**Comment by PlagueCZ** [comment]
Ah sorry, missing round count, so the defaults for arguments or requiring them will fix this for everyone in the future.

**Comment by PlagueCZ** [comment]
What is interesting is that if I run `iperf3` manually without `-M` I get 42Gb/s, if I use `-M` I only get the same number as your script, which is 27Gb/s And I cannot for the life of me find out the default value used there.

File: hack/connectivity_test/flow_test.py [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+	parser_server = subparsers.add_parser('server', help='Run as an iperf3 server')
+	add_shared_subparser_args(parser_server)
+	parser_server.set_defaults(func=test_server)
+
+	# Create the parser for the "client" command
+	parser_client = subparsers.add_parser('client', help='Run as an iperf3 client')
+	add_shared_subparser_args(parser_client)
+	parser_client.add_argument('--run-time', type=int, help="Duration (in seconds) for each iperf3 test.")
+	parser_client.add_argument('--payload-length', type=int, help="Specify the maximum payload length of a tcp packet.")
+	parser_client.add_argument('--round-count', type=int, help="Number of test rounds to execute.")
```
[the remark this answers is not in the document]
Comment (byteocean) [reply hack/connectivity_test/flow_test.py]: enhanced.

**Review by PlagueCZ — APPROVED** [review]
Works locally on my machine, results are consistent with manual observations.

**Review by guvenc — APPROVED** [review]
Thanks. I could also run the tests successfully in my environment. Looks good.

**Pull request merged**